src/index.ts

```diff
@@ -56,6 +56,7 @@
  * region_tag:storage_quickstart
  * Full quickstart example:
  */
+export * from './nodejs-common';
```

Comment:
Do you think we should export everything from common? It seems like most things have no real use outside of storage and I'm nervous about making it all publicly exposed.

Reply:
Fair - exporting `ApiError` only